puppet/modules/profiles/manifests/redmine.pp

```diff
@@ -1,8 +1,6 @@
 # @summary A Redmine server
 class profiles::redmine (
 ) {
-  # Redmine is already included via ENC
-  # TODO include redmine
   $backup_path = '/var/lib/redmine'
```

Comment:
My original was:
```puppet
include redmine
$backup_path = $redmine::data_dir
```
Perhaps I should have added that second line as TODO as well, but can you do it now?